### PR TITLE
Wrap package for CppUTest 4.0

### PR DESCRIPTION
**Pull request by dreamer-coding**
Wrap package for CppUTest. There are two dependencies declared in the package one for the test framework and the other for the framework extensions.

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+cpputest_dirs = include_directories('include')
+subdir('src')
+
+cpputest_dep = declare_dependency(
+    link_with : cpputest_lib,
+    version : meson.project_version(),
+    include_directories : cpputest_dirs)
+
+if get_option('extinctions') == true
```
Comment (legeana) [inline meson.build]: `if X == true` can generally be rewritten as `if X` I believe and it looks much better.

File: src/meson.build [hunk 2]
```diff
@@ -0,0 +1,7 @@
+if get_option('extinctions') == true
```
Comment (legeana) [inline src/meson.build]: Same, please consider using `if get_option('...')` without `== true`.

File: upstream.wrap [hunk 3]
```diff
@@ -0,0 +1,6 @@
+[wrap-file]
+directory = cpputest
```
Comment (legeana) [inline upstream.wrap]: I believe it should be `cpputest-4.0` based on the directory name in the archive.

**Review by legeana — CHANGES_REQUESTED** [review]

**Comment by dreamer-coding** [comment]
Sure I will get on that

**Comment by dreamer-coding** [comment]
Also fixed.

**Pull request merged**